content/enterprise_influxdb/v1.9/administration/config-data-nodes.md

```diff
@@ -1251,3 +1252,33 @@ Environment variable: `INFLUXDB_TLS_MAX_VERSION`
     > If AE is left enabled while lowering your `max-fetch`, you will initially see
     > higher CPU load as new shard digest files are created.
     > The added load will drop off after shard digests are completed for existing shards. -->
+
+## Flux query settings
+
+### `[flux-controller]`
+
+#### query-concurrency
+Number of queries allowed to execute concurrently.
+`0` means unlimited.
+Default is `0`.
+
+#### query-initial-memory-bytes
+Initial bytes of memory allocated for a query.
+`0` means unlimited.
+Default is `0`.
+
+#### query-max-memory-bytes
+Maximum total bytes of memory allowed for queries.
+`0` means unlimited.
+Default is `0`.
+
+#### total-max-memory-bytes
+Maximum total bytes of memory allowed for queries.
```

Comment:
Might be good to clarify - this is the max memory usage across all running flux queries, while `query-max-memory-bytes` is the per-query maximum.